akka-docs/src/main/paradox/scala/fault-tolerance.md

```diff
@@ -73,10 +89,14 @@ exceptions are handled by default:
 If the exception escalate all the way up to the root guardian it will handle it
 in the same way as the default strategy defined above.
 
+@@@ div { .group-scala }
+
 You can combine your own strategy with the default strategy:
 
 @@snip [FaultHandlingDocSpec.scala]($code$/scala/docs/actor/FaultHandlingDocSpec.scala) { #default-strategy-fallback }
 
+@@@
```

Comment:
Combining with default strategy requires `applyOrElse`, which is only suitable in Scala.

![image](https://user-images.githubusercontent.com/7414320/27213999-1123c4a8-52a5-11e7-83b8-be2b9085c9b8.png)